### PR TITLE
Fix to retain selected network interface on refresh.

### PR DESCRIPTION
**Pull request by dwoodard1**
This PR is for #290

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Network/NetworkInterfacesTableUi.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
 
 	private static final Messages MSGS = GWT.create(Messages.class);
 	private final GwtNetworkServiceAsync gwtNetworkService = GWT.create(GwtNetworkService.class);
+	private final String SELECTED_INTERFACE = "ui.selected.interface";
```
Comment (MMaiero) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Network/NetworkInterfacesTableUi.java]: @dwoodard1 Maybe this variable could be static final

**Comment by cdealti** [comment]
@dwoodard1 It works for me. Ready to merge if you accept @MMaiero suggestion

**Comment by dwoodard1** [comment]
@MMaiero I am not sure about the static variable. I think we want a separate instance for each instantiation. In case you are working with the Kura UI with separate tabs/windows/browsers.

**Comment by dwoodard1** [comment]
@MMaiero Made your suggested change, have a look.

**Pull request merged**